script/test

```diff
@@ -1,11 +1,16 @@
 #!/bin/bash
 #
-# Runs all tests.
+# Runs all tests & lint.
 
 set -euo pipefail
 source "$(dirname "$0")/utilities.bash"
 
 header "Tests"
-govendor test -race -v +local
+go test -v $(glide novendor)
```

Comment:
- [ ] Omit `-v`.  
It adds noise to paw through when there's an actual test failure.